docs/source/setup-guide.rst

```diff
@@ -79,6 +79,11 @@ This starts a subshell; you can leave the environment by typing ``exit``.
 Later when you're building a project bitstream, 
 you need to add USE\_SYMBIFLOW=1 to your make command line.
 
+If your command does not succeed you may need to lower the target clock rate.
```

Comment:
I think we need to be a little bit more precise here, since the confusing thing is that the bitstream generation did not give a clear error, so you didn't see any problem until you tried to run the software on the bitstream on the board.  Maybe "If you see no errors, but the compiled RISC-V executable does not run on the board, you may need to lower the target clock rate" -- sound ok?

Reply:
Thanks for the suggestion, I agree - have pushed an update.